### ISSUE TITLE
 Math is unreadable in iPhone lockdown mode

### ISSUE THREAD

**Issue by pjj** [arXiv/html_feedback#201]
### Description

As per the title, the math looks incomprehensible in lockdown mode of an iPhone.  

### (Required) What is this issue most closely related to? Select one.

Formula

### Internal issue ID

50c4d7c1-e9b1-4f18-aeb3-af69274e4c66

### Paper URL

https://browse.arxiv.org/html/2312.14680v1

### Browser

Safari/605.1.15

### Device Type

Desktop

**Comment by dginev** [comment]
Hi @pjj , is that problem still encountered in 2026? MathML support has been improving in the last couple of years, and I would hope the "lockdown mode" resolved whatever issues it had back then.

**Comment by pjj** [comment]
Yes there are still problems with math display on the above paper.  It feels less messy than in the past, but it is still unusable for serious reading.

**Comment by dginev** [comment]
@pjj It would be a great help if you could point to a couple of examples on the page and explain the breakage you see.

I can take it from there to infer the remaining problems. Thank you in advance!

**Comment by pjj** [comment]
Here are 3 snapshots from parts of the paper in case it helps.  

![image](https://github.com/user-attachments/assets/17c8f30c-b74f-42cb-9284-c90081a72c73)

![image](https://github.com/user-attachments/assets/6ee7228b-f38e-45c0-9423-7ccd3d4d76f8)

![image](https://github.com/user-attachments/assets/f619f4e4-e49f-4ccf-923b-0a71561ac350)

**Comment by dginev** [comment]
Thank you - this is extremely helpful.

It appears that the MathML rendering in your browser is not even starting. I'll grab an iPhone user from our team and investigate.
